fastlane/lib/fastlane/plugins/template/lib/fastlane/plugin/%plugin_name%/actions/%plugin_name%_action.rb.erb

```diff
@@ -6,11 +6,11 @@ module Fastlane
       end
 
       def self.description
-        %q{<%= summary %>}
+        '<%= summary %>'
```

Comment:
This won't work when the user adds a `\` in their description, but even then they'll get an appropriate Ruby error when trying to run it